### ISSUE TITLE
If an AMD Module nests a return statement below the conditionals, then the return doesn't become an export.

### ISSUE THREAD

**Issue by stevethedev** [buxlabs/amd-to-es6#103]
**Target Code**
```javascript
define([],function(){
    function a(){}
    if ("function"===typeof Object.create)
        return Object.create;
    else
        return function(b){a.prototype=b;var c=new a;a.prototype=null;return c};
});
```

**Expected Result**
```javascript
function a() {}
export default ("function" === typeof Object.create)
    ? Object.create
    : function(b){a.prototype=b;var c=new a;a.prototype=null;return c};
```

**Actual Result**
```javascript
function a() {
}
if ('function' === typeof Object.create)
    return Object.create;
else
    return function (b) {
        a.prototype = b;
        var c = new a();
        a.prototype = null;
        return c;
    };
```

**Comment by emilos** [comment]
@stevethedev thx, I'll investigate. In the meantime, as a workaround, you could assign the result of the if statement to a variable so that there is one return.

**Comment by pkonieczniak** [comment]
It has been fixed in https://github.com/buxlabs/amd-to-es6/commit/6f3ab08c7e1d424d14fe157e1c87327d9bb0025e and released as 0.12.5. 
Thanks !
